database-api/src/main/kotlin/fi/vauhtijuoksu/vauhtijuoksuapi/database/api/VauhtijuoksuDatabase.kt

```diff
@@ -23,6 +23,14 @@ interface VauhtijuoksuDatabase<T : Model> {
      * Update given record and return the updated record if it exists. Note that full record is expected as input,
      * i.e. any null values are written to db.
      */
+
+    fun addMany(records: List<T>): Future<List<T>?> { return Future.future { p -> p.complete(listOf()) } }
```

Comment:
Make the default implementation throw instead

Reply:
thrown